libs/vkd3d/command.c

```diff
@@ -4862,27 +4866,22 @@ static void STDMETHODCALLTYPE d3d12_command_list_OMSetRenderTargets(d3d12_comman
         if (!rtv_desc || !rtv_desc->resource)
         {
             WARN("RTV descriptor %u is not initialized.\n", i);
-            list->rtvs[i] = VK_NULL_HANDLE;
             continue;
         }
 
         d3d12_command_list_track_resource_usage(list, rtv_desc->resource);
 
         /* In D3D12 CPU descriptors are consumed when a command is recorded. */
-        view = rtv_desc->view;
-        if (!d3d12_command_allocator_add_view(list->allocator, view))
-        {
+        if (!d3d12_command_allocator_add_view(list->allocator, rtv_desc->view))
```

Comment:
Hm, what is actually happening here, do we create a new VkImageView for every OMSetRenderTargets?

Reply:
We don't, `add_view` only ref-counts the view. Not sure if this is really necessary, but it was there before so I kept it.

Reply:
I assume we need it if the descriptors are meant to be consumed/copied in CPU timeline.